controllers/recipes.js

```diff
@@ -31,40 +31,44 @@ module.exports = (app) => {
   })
 
   function pullEdamamRecipes() {
-    // TODO: add loop later to change from/to params + add max (currently 525 keto recipes)
-    const url = `https://api.edamam.com/search?q=keto&from=0&to=100&app_id=${EDAMAM_APP_ID}&app_key=${EDAMAM_API_KEY}`;
+    let start;
+    const increment = 100;
+    for (start = 0; start <= 600; start += increment) {
+      const url = `https://api.edamam.com/search?q=keto&from=${start}&to=${start+increment}&app_id=${EDAMAM_APP_ID}&app_key=${EDAMAM_API_KEY}`;
 
-    http.get(url, (response) => {
-      response.setEncoding('utf8');
-      let body = '';
-      response.on('data', (d) => { body += d });
+      http.get(url, (response) => {
+        response.setEncoding('utf8');
+        let body = '';
+        response.on('data', (d) => { body += d });
 
-      response.on('end', () => {
-        const parsed = JSON.parse(body);
+        response.on('end', () => {
+          const parsed = JSON.parse(body);
+          console.log(parsed.count);
```

Comment:
i'm not sure if these logs were intentional or for debugging but if you want to use these logs on the server, it'd be helpful to add a string to the beginning so you know what is being spit out.

Reply:
good catch! fixed that.